bin/morgue.js

```diff
@@ -5067,6 +5068,24 @@ function coronerNuke(argv, config) {
     errx('No such object.');
   }
 
+  if (!argv["-f"]) {
+    const name = target.get('name');
+    const rl = readline.createInterface({
+      input: process.stdin,
+      output: process.stdout,
+      terminal: true
+    });
+    await new Promise(done => {
+      rl.question("You are about to nuke " + name + ". Please type '" + name +"' to confirm: ", confirmation => {
```

Comment:
In former projects I've used short random nonce for this kind of thing, but I suppose adding an "echo $name" to your script really means you mean it.  Maybe to make it super clear in such scripts, consider asking for "really nuke $name" or something like that, like AWS.

Reply:
AWS, to my knowledge, either requires strictly the `name` or the word `delete` in some cases